### PR TITLE
feat(notifications): Notification Center

### PR DESCRIPTION
**Pull request by kduralek**

### CODE REVIEW

File: src/app/marketplace/marketplace-my-files.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import { ReadyToDownloadService } from './services/ready-to-download.service';
+
+describe('MarketplaceMyFilesComponent', () => {
+  let component: MarketplaceMyFilesComponent;
+  let fixture: ComponentFixture<MarketplaceMyFilesComponent>;
+
+  beforeEach(async(() => {
+    const dataProduct = new DataProduct();
+    const myActiveListingsServiceSpy = jasmine.createSpyObj('MyActiveListingsService', [ 'getProducts' ]);
+    myActiveListingsServiceSpy.getProducts.and.returnValue(new BehaviorSubject<DataProduct[]>([ dataProduct ]));
```
Comment (davo22) [inline src/app/marketplace/marketplace-my-files.component.spec.ts]: Function getProducts returns Observable, not a Subject.

File: src/app/marketplace/marketplace-my-files.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    const readyToDownloadServiceSpy = jasmine.createSpyObj('ReadyToDownloadService', [ 'getProducts' ]);
+    readyToDownloadServiceSpy.getProducts.and.returnValue(new BehaviorSubject<DataProduct[]>([ dataProduct ]));
```
Comment (davo22) [inline src/app/marketplace/marketplace-my-files.component.spec.ts]: As above

File: src/environments/environment.template.ts [hunk 3]
```diff
@@ -37,6 +37,7 @@ export const environment = {
       defaultName: 'REPUX',
       precision: 18,
       format: '1.2-18',
+      shortFormat: '1.2-4',
```
Comment (davo22) [inline src/environments/environment.template.ts:40]: I think that truncating prices is bad idea. When real price is e.g. 0,00001 then we display 0,0000. If you want to truncate price, you should first check if you not changing value.

File: src/styles.scss [hunk 4]
```diff
@@ -32,7 +32,7 @@ app-root {
 .mat-stroked-button,
 .mat-raised-button {
   text-transform: uppercase;
-  border-radius: 4px;
+  border-radius: 4px !important;
```
Comment (davo22) [inline src/styles.scss:35]: Please change selector to be more accurate to avoid using !important.

**Review by davo22 — APPROVED** [review]

File: src/styles.scss (hunk 4 above)
Comment (kduralek) [reply src/styles.scss:35]: That's exact the same selector as in Angular Material. In order to override buttons without `!important` we need to add selectors like `.mat-flat-button.mat-primary` `.mat-flat-button.mat-accent` and for every colour available. I would rather use `!important` than compound selectors.

File: src/environments/environment.template.ts (hunk 3 above)
Comment (kduralek) [reply src/environments/environment.template.ts:40]: Value is rounded - as per request from Ola :)

**Pull request merged**